tutorials/08-tdem/plot_fwd_1_em1dtm.py

```diff
@@ -0,0 +1,175 @@
+"""
+1D Forward Simulation for a Single Sounding
+===========================================
+
+Here we use the module *SimPEG.electromangetics.time_domain_1d* to predict
+the stepoff response for a single sounding over a 1D layered Earth.
+In this tutorial, we focus on the following:
+
+    - Defining receivers, waveforms, sources and the survey
+    - The units of the model and resulting data
+    - Defining and running the 1D simulation for a single sounding
+
+Our survey geometry consists of a horizontal loop source with a radius of 6 m
+located 20 m above the Earth's surface. The receiver is located at the centre
+of the loop and measures the vertical component of the response.
+
+
+"""
+
+#####################################################
+# Import Modules
+# --------------
+#
+
+import numpy as np
+import os
+from matplotlib import pyplot as plt
+from discretize import TensorMesh
+
+from SimPEG import maps
+import SimPEG.electromagnetics.time_domain as tdem
+from SimPEG.electromagnetics.utils.em1d_utils import plot_layer
+
+write_output = False
+plt.rcParams.update({'font.size': 16})
+
+# sphinx_gallery_thumbnail_number = 2
+
+#####################################################################
+# Create Survey
+# -------------
+#
+# Here we demonstrate a general way to define the receivers, sources, waveforms and survey.
+# For this tutorial, the source is a horizontal loop whose current waveform
+# is a unit step-off. The receiver measures the vertical component of the magnetic flux
+# density at the loop's center.
+#
+
+# Source properties
+source_location = np.array([0., 0., 20.])  
+source_orientation = "z"                      # "x", "y" or "z"
+source_current = 1.                           # maximum on-time current
+source_radius = 6.                            # source loop radius
+
+# Receiver properties
+receiver_location = np.array([0., 0., 20.])
+receiver_orientation = "z"                    # "x", "y" or "z"
+times = np.logspace(-5, -2, 31)               # time channels (s)
+
+# Define receiver list. In our case, we have only a single receiver for each source.
+# When simulating the response for multiple component and/or field orientations,
+# the list consists of multiple receiver objects.
+receiver_list = []
+receiver_list.append(
+    tdem.receivers.PointMagneticFluxDensity(
```

Comment:
For this and all of the following tdem tutorials, I would suggest that we use the `PontMagneticFluxDensityTimeDerivative` receivers. It is much more common to work with db/dt data than it is to work with B data in a tdem experiment

Reply:
True. I did not realize that. @dccowan: what's your thought? If you agree, can you update the tutorial?

Reply:
just pinging again on this @sgkang, @dccowan: I think for the TDEM examples, using the Time Derivative is much more common